app/ui/browser-modern/constants/effect-types.js

```diff
@@ -44,6 +44,8 @@ export const SET_PAGE_ZOOM_LEVEL = 'SET_PAGE_ZOOM_LEVEL_EFFECT';
 export const CAPTURE_PAGE = 'CAPTURE_PAGE_EFFECT';
 export const PARSE_PAGE_META_DATA = 'PARSE_PAGE_META_DATA_EFFECT';
 export const GET_CERTIFICATE_ERROR = 'GET_CERTIFICATE_ERROR_EFFECT';
+export const PIN_TAB = 'PIN_TAB';
+export const UNPIN_TAB = 'UNPIN_TAB';
```

Comment:
By convention, effect action string identifiers have an `_EFFECT` suffix so that they're more easily distinguished in console logs.